lib/job_tomate/actions/jira_update_issue_assignee_and_roles_for_status.rb

```diff
@@ -116,10 +116,13 @@ def update_assignee_and_role(issue, user, role)
 
       def unassign(issue)
         update_issue(issue, fields: { assignee: nil })
+        issue.change_assignee_name(nil)
```

Comment:
i've just googled CQRS, haven't seen it before :) Looks curious!
i'd like 5mins to discuss how to best handle the changes brought by the `actions` module (like issue re-assignment) -- and who exactly should be notified on potential problems like missing bug cause. Should it always be the assignee no matter the issue status or always the assigned developer?